source/includes/add-new-app.md

```diff
@@ -0,0 +1,36 @@
+# Add a New App
+
+Tell us about an app we don't know about yet.
+We do our best to find all apps, but occasionally we miss one (there isn't a complete list for Google Play).
+
+### HTTPS Request
+
+`https://api.appmonsta.com/v1/stores/<store>/details/<app_id>.json`
+
+### Request Method: PUT
+
+### Request Parameters
+
+Parameter         | Required | Value
+----------------- | -------- | -----------
+**store**         | Yes      | `android` or `itunes`.
+**app_id**        | Yes      | the id of the app to add, e.g. com.rovio.angrybirds.
+
+### Response: 202 if successfully added
+
+Check right side shell for such example.
+
+```
+$ curl -i -s -u '<your_api_key>:X' https://api.appmonsta.com/v1/stores/android/details/com.rovio.json -X PUT
```

Comment:
Use the same "template" for API key that we use elsewhere. The same goes to the whole structure (if it applies anywhere).